examples/deploy-with-sagemaker/deploy-with-sagemaker.ipynb

```diff
@@ -0,0 +1,337 @@
+    "    def predict(self, series):\n",
+    "        \"\"\"\n",
+    "        predict expects pandas.Series as input\n",
+    "        \"\"\"        \n",
+    "        return self.artifacts.sentiment_lr.predict(series)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sentiment_lr_model import SentimentLRModel\n",
+    "\n",
+    "# Initialize bentoML model with artifacts\n",
+    "\n",
+    "bento_model = SentimentLRModel.pack(\n",
+    "    sentiment_lr=sentiment_lr\n",
+    ")\n",
+    "\n",
+    "# Save bentoML model to directory\n",
+    "saved_path = bento_model.save(\"/tmp/bento\")\n",
+    "\n",
+    "# print the directory containing exported model archive (prefixed with model name and version)\n",
+    "print(saved_path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Load model from archive"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "import bentoml\n",
+    "\n",
+    "# Load exported bentoML model archive from path\n",
+    "bento_model = bentoml.load(saved_path)\n",
+    "\n",
+    "# Call predict on the restored sklearn model\n",
+    "bento_model.predict(pd.Series([\"hello\", \"hi\"]))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### * For demo purpurse, copy generated model to ./model folder"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import shutil\n",
+    "shutil.rmtree('./model', ignore_errors=True)\n",
+    "shutil.copytree(saved_path, './model')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Deploy to AWS SageMaker\n",
+    "\n",
+    "```bash\n",
+    "bentoml deploy ARCHIVE_PATH --platform=PLATFORM --region=REGION --stage=STAGE\n",
+    "```\n",
+    "### Arguments:\n",
+    "* archive_path: The file path or s3 that contains BentoML bundles.\n",
+    "\n",
+    "### Options:\n",
+    "* platform: REQUIRED.  The platform that you want to deploy bentoml bundle to.  For serverless, we support aws-lambda, aws-lambda-py2 and gcp-function.\n",
+    "* region: OPTIONAL The cloud provider's region you want to deploy in.\n",
+    "* stage: OPTIONAL Stage is a helper value that identify the position in the workflow process. DEFAULT value is dev\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!bentoml deploy ./model --platform aws-sagemaker --region us-west-2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Check deployment status\n",
+    "\n",
+    "```\n",
+    "bentoml checkt-deployment-status ARCHIVE_PATH --platform=PLATFORM --region=REGION --stage=STAGE\n",
```

Comment:
similarly in the notebook,  add curl request example

Reply:
done